tests/phpunit/tests/theme/wpAddGlobalStylesForBlocks.php

```diff
@@ -75,6 +75,84 @@ public function test_third_party_blocks_inline_styles_get_registered_to_global_s
 		);
 	}
 
+	/**
+	 * Ensure that the block cache is set for global styles.
```

Comment:
```suggestion
	 * Tests that the block cache is set for global styles.
```
Nit: Tests usually begin with "Tests that", and this aligns with the use of third-party singular verbs used in method docblocks. [Ref](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#documenting-tips)

:1234: Applies to other tests in the PR.

Reply:
Happy to make that change to the included tests. Also updated the ticket number now that this is being associated by a new Trac ticket.

See: [6c35708](https://github.com/WordPress/wordpress-develop/pull/6879/commits/6c35708ef008b468b9afac9e2648ebc5d9af71dc)